### PR TITLE
Work mode and process proposal

### PR DESCRIPTION
**Pull request by brentzundel**
This PR alters the contributing.md document to more closely reflect the work mode currently followed by the group and to add clarity around the process of changing spec status and publishing.
Signed-off-by: Brent Zundel <brent.zundel@gmail.com>

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -1,126 +1,179 @@
...
+1. Consensus should be easy - this means that if any group participant feels
+  strongly that an issue should be included in the next version of the
+  specification, then it should be included, otherwise it may be included in
+  some future version.
+1. An expectation of doing - this means that any group participant who feels
+  strongly that an issue should be included in the next version of the
+  specification should also be the one to propose concrete changes and work to
+  bring the group to consensus about the changes.
+
+#### Speciation Status and Publishing
```
Comment (OR13) [inline contributing.md:62]:
```suggestion
#### Specification Status and Publishing
```

File: contributing.md [hunk 2]
```diff
@@ -1,126 +1,179 @@
...
+  on that version (in accordance with the DIF Charter Deliverable Development
+  Process).
+1. `Working Group Approved` - Once the Working Group has refined the `Draft
+  Deliverable` and believes it has addressed all pertinent issues, it may
+  progress the `Draft Deliverable` to `Working Group Approved` status (in
+  accordance with the DIF Charter Deliverable Development Process). 
+  
+A specification with the `Working Group Approved` status indicates that all of
+the following conditions are met: 
+   * There is quality code that demonstrates the concepts it describes.
```
Comment (OR13) [inline contributing.md:82]: Might be good to specify test vectors here as well, while code it nice, test vectors encourage implementations in other languages, which will further support the standard. 

The first implementation should be used to generate the test vectors, using as much determinism as possible.

**Review by OR13 — CHANGES_REQUESTED** [review]
I'd like to see test vectors as a requirement for "Working Group Approved".

File: contributing.md [hunk 3]
```diff
@@ -1,126 +1,179 @@
...
+represent the diversity of the Working Group (i.e., no two maintainers may work
+for the same company, organization, or project).
+
+## Work Mode and Process
+
+### Process Summary
+The following is a summary of the process used by the Claims and Credentials
+Working Group for the Presentation Exchange Specification. Greater detail may
+be found below.
+1. The group determines the version number of the next release of the
```
Comment (glcohen) [inline contributing.md:24]: All 1?

Comment (brentzundel) [reply contributing.md:24]: markdown should render them as ordered numerical bullets, so I am in the habit of always using 1s so I never have to re-number bullet points when adding to or reconfiguring numbered lists

File: contributing.md (hunk 2 above)
Comment (brentzundel) [reply contributing.md:82]: Something like this?
```suggestion
   * There is quality code that demonstrates the concepts it describes.
   * There are test vectors which may be used to test implementations.
```

Comment (OR13) [reply contributing.md:82]: Yep, I would approve with that change.

**Review by glcohen — APPROVED** [review]

**Review by glcohen — APPROVED** [review]

File: contributing.md (hunk 2 above)
Comment (glcohen) [reply contributing.md:82]: @OR13 can you approve?

**Review by OR13 — APPROVED** [review]

**Pull request merged**